includes/scheduler/class-post.php

```diff
@@ -62,7 +63,7 @@ public static function schedule_post_activity( $post_id, $post, $update, $post_b
 				break;
 
 			case 'trash':
-				$type = 'Delete';
+				$type = 'federated' === get_wp_object_state( $post ) ? 'Delete' : false;
```

Comment:
`get_wp_object_state` is a function I kept for backwards compatibility and it feels no longer valid. we shouldn't rely to heavily on it, but think about a proper alternative, that fits better with the new outbox flow.